app/models/vita_partner.rb

```diff
@@ -50,12 +51,9 @@ class VitaPartner < ApplicationRecord
   def at_capacity?
     return parent_organization.at_capacity? if site?
 
-    return false if capacity_limit.blank?
+    return false if capacity_limit.blank? || organization_capacity.active_client_count.nil?
 
-    Client
-      .where(vita_partner_id: [id, *child_site_ids])
-      .joins(:tax_returns).where(tax_returns: { status: TaxReturnStatus::STATUS_KEYS_INCLUDED_IN_CAPACITY })
-      .count >= capacity_limit
+    return organization_capacity.active_client_count >= capacity_limit
```

Comment:
Replacing this query with the result of our view! Had a discussion with Ben Sheldon about how active record is able to "drop down" the query for this one vita partner into the view code so that it only queries for the vita partner we want. super cool!